lib/project_types/script/graphql/app_script_update_or_create.graphql

```diff
@@ -1,26 +1,30 @@
 mutation AppScriptUpdateOrCreate(
   $extensionPointName: ExtensionPointName!,
   $title: String,
-  $configUi: String,
   $sourceCode: String,
   $language: String,
   $force: Boolean,
   $schemaMajorVersion: String,
   $schemaMinorVersion: String,
   $useMsgpack: Boolean,
-  $uuid: String
+  $uuid: String,
+  $configurationUi: Boolean,
+  $configurationDefinitionVersion: String,
```

Comment:
Do you think we should rename this to `scriptJsonVersion` before shipping everything? If so, I can put up a Script Service PR to update it.

Reply:
Ah yeah its probably a good idea to do it now!